airbyte-integrations/connectors/source-goldcast-api/source_goldcast_api/schemas/agenda_items.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Agenda Items Schema",
+  "type": "array",
```

Comment:
This should always be an object instead of array.
Add `additionalProperties: true`

Reply:
This is more an internal requirement to make schema more flexible for future updates in the API.

Reply:
@marcosmarxm , I agree, I am referring to the comment on the array vs object. I think the API response is that of an array, not an object